nextflow_schema.json

```diff
@@ -435,28 +241,126 @@
             "$ref": "#/definitions/input_output_options"
         },
         {
-            "$ref": "#/definitions/basecalling_demultiplexing_options"
+            "$ref": "#/definitions/reference_genome_options"
         },
         {
-            "$ref": "#/definitions/alignment_options"
+            "$ref": "#/definitions/institutional_config_options"
         },
         {
-            "$ref": "#/definitions/differential_analysis_options"
+            "$ref": "#/definitions/max_job_request_options"
         },
         {
-            "$ref": "#/definitions/process_skipping_options"
+            "$ref": "#/definitions/generic_options"
+        }
+    ],
+    "properties": {
+        "protocol": {
+            "type": "string"
         },
-        {
-            "$ref": "#/definitions/reference_genome_options"
+        "input_path": {
+            "type": "string"
         },
-        {
-            "$ref": "#/definitions/generic_options"
+        "flowcell": {
+            "type": "string"
         },
-        {
-            "$ref": "#/definitions/max_job_request_options"
+        "kit": {
+            "type": "string"
         },
-        {
-            "$ref": "#/definitions/institutional_config_options"
+        "barcode_kit": {
+            "type": "string"
+        },
+        "barcode_both_ends": {
+            "type": "string"
+        },
+        "guppy_config": {
+            "type": "string"
+        },
+        "guppy_model": {
+            "type": "string"
+        },
+        "guppy_gpu": {
+            "type": "string"
+        },
+        "guppy_gpu_runners": {
+            "type": "integer",
+            "default": 6
```

Comment:
These all need to be added properly to the Schema via `nf-core schema build`. Looks like I had already added descriptions, logos already but they were deleted?